generators/generator-base.js

```diff
@@ -1412,7 +1422,7 @@ Generator.prototype.generateKeyStore = function() {
             `-dname "CN=Java Hipster, OU=Development, O=${this.packageName}, L=, ST=, C="`
         , function(code) {
             if (code !== 0) {
-                parent.env.error(chalk.red(`\nFailed to create a KeyStore with \'keytool\'`), code);
+                parent.env.error(chalk.red('\nFailed to create a KeyStore with \'keytool\''), code);
```

Comment:
we should have used `parent.error` here without needing to use chalk here. could you plz change it as part of this PR